com.ibm.streamsx.topology/opt/python/include/splpy_general.h

```diff
@@ -788,6 +835,16 @@ class SplpyGeneral {
         }
         return pySet;
     }
+ 
+#ifdef SPL_RUNTIME_TYPE_OPTIONAL_H 
+    // SPL optional tyoe to Python optional tyoe
```

Comment:
It's to a Python object, not a Python optional type, since there's no such thing.